monai/transforms/utils.py

```diff
@@ -103,6 +106,10 @@
     "generate_spatial_bounding_box",
     "get_extreme_points",
     "get_largest_connected_component_mask",
+    "get_largest_connected_component_mask_point",
+    "sample_points_from_label",
+    "erode3d",
```

Comment:
discussed offline with @KumoLiu , I will check if this transform can use maisi's version

Reply:
https://github.com/Project-MONAI/MONAI/blob/6c23fd06fc11667beedd0ba730d4104076a8db2d/monai/apps/generation/maisi/utils/morphological_ops.py#L24

Would also be better to modify the utils place?

Reply:
thanks @KumoLiu . I checked the code, it can be replaced by existing morphological function. One question:
https://github.com/Project-MONAI/MONAI/blob/410109a8dab9b244e79697ee89e13c8b044da6dd/monai/apps/generation/maisi/utils/morphological_ops.py#L113

is it an error? should it be `torch.ones((mask_t.shape[0], mask_t.shape[1]) + filter_size)` ?

Reply:
I checked the code, we do can use maisi's version

Reply:
so far, `MONAI/monai/apps/generation/maisi/utils/morphological_ops.py` is not used by any other files, can we move all of its content into `MONAI/monai/transforms/utils.py` to make it more generic?

Hi @ericspod @Can-Zhao , what do you think?

Reply:
> so far, `MONAI/monai/apps/generation/maisi/utils/morphological_ops.py` is not used by any other files, can we move all of its content into `MONAI/monai/transforms/utils.py` to make it more generic?
> 
> Hi @ericspod @Can-Zhao , what do you think?

We had discussed earlier about moving morphological operations into a utilities place in core and refactoring anything needing such operations to use these. We could move `morphological_ops.py` to `monai/transforms` or somewhere else making sense, the `utils.py` file is way too large already.

Reply:
HI @ericspod , thanks for the explain. As for
> refactoring anything needing such operations to use these
May I know where should changed? In MONAI tutorials?

Reply:
I think there are other areas in core that are using morphological operations, I haven't had a chance to look but we can come back to this if we have generic implementations we can used in utils.

Reply:
Thanks, I checked core and tutorials, and modify other areas in core. I will update the tutorial after this PR is merged.